### DISCUSSION TITLE
[1.3.0] Tasha's Cauldron of Everything Changelog

### DISCUSSION THREAD

**Discussion by mattexdee** [foundryvtt/foundryvtt-premium-content#685] [Updates]
## Feature Enhancements
- This update brings tattoos, their data models, and sheets up to date with the official 4.0.0 game system and includes activities and new app V2 styling.
- It also improves the source field data for all items and features so they appear more nicely in the Compendium Browser.

**Comment by itteerde** [comment]
We need separate issue tracking for the products, at the very least per system (please per product).

**Reply by mattexdee** [reply]
There is separate issue tracking for each product. When you [submit a bug](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml&title=%5BBug%5D%3A+>) you'll be prompted for the product that you're having an issue with and the version. A label will be automatically applied to it for easy searching. For example, this link has all [Tasha's Issues](<https://github.com/foundryvtt/foundryvtt-premium-content/issues?q=is%3Aopen+is%3Aissue+label%3Adnd-tashas-cauldron>) and this one has all issues for the new [Player's Handbook](<https://github.com/foundryvtt/foundryvtt-premium-content/issues?q=is%3Aopen+is%3Aissue+label%3Adnd-players-handbook>).

Changelogs can also be filtered using the same tags. For example, here are all [Tasha's changelogs](<https://github.com/foundryvtt/foundryvtt-premium-content/discussions?discussions_q=is%3Aopen+label%3Adnd-tashas-cauldron>).
